_episodes/13-second-welcome.md

```diff
@@ -17,20 +17,33 @@ keypoints:
 - "Teaching is a learned skill."
 ---
 
-Yesterday we focused on general aspects of educational psychology and pedagogy. Today we will focus on the specifics of teaching Software Carpentry, Data Carpentry, and Library Carpentry workshops.
-We will look at the lessons that have been written by the Carpentries and discuss how to teach effective workshops. We will review specific
-teaching practices we follow at our workshops and practice using some of them. One of the most important practices is participatory live coding. You rarely
-find this approach in a current university setting so we'll spend some time practicing this skill. We will conclude our training with a discussion 
-about workshop logistics that will help you in preparing to teach your first workshop.  
+To recap from yesterday's introduction, the goal of this training is prepare
+you to be a certified Carpentries instructor, which includes the following:
+- be familiar with and understand how to apply research-based teaching principles,
+especially as they apply to the Carpentries audience.
+- understand the importance of a respectful and inclusive classroom environment; commit to
+creating such an environment; and be able to
+identify and implement Carpentries policies and general practices to accomplish this.
+- practice and develop skills in the teaching methods used in Carpentries workshops
+- learn enough about the Carpentries organization to know where to go for help,
+how to start organizing a workshop, and how to get involved with community activities.
 
+Yesterday we focused on general aspects of educational psychology and pedagogy and
+how to create a positive classroom as environment, addressing the first two goals.
+We started to address the third goal by discussing the important of lesson study
+and observation, and by providing an opportunity to teach and
+receive feedback. We will continue that skill development process today, as we
+focus on the specifics of teaching Software Carpentry, Data Carpentry, and Library Carpentry workshops.
+We will review specific
+teaching practices we follow at our workshops and practice using some of them. One of the most important practices is participatory live coding. You rarely
+find this approach in a current university setting so we'll spend some time practicing this skill.
```

Comment:
There are a few takes on why we practice -- I see these noted elsewhere but here might be good to prepare them to think in all the ways. One of them is the rehearsal of live coding itself. Others are that we are rehearsing the process of reflective practice, as well as developing skills in providing constructive feedback.

Reply:
good point. I'll add those aspects as well.